nautobot/extras/forms.py

```diff
@@ -483,6 +486,7 @@ class ConfigContextFilterForm(BootstrapMixin, forms.Form):
     region = DynamicModelMultipleChoiceField(queryset=Region.objects.all(), to_field_name="slug", required=False)
     site = DynamicModelMultipleChoiceField(queryset=Site.objects.all(), to_field_name="slug", required=False)
     role = DynamicModelMultipleChoiceField(queryset=DeviceRole.objects.all(), to_field_name="slug", required=False)
+    type = DynamicModelMultipleChoiceField(queryset=DeviceType.objects.all(), to_field_name="slug", required=False)
```

Comment:
Maybe also add to `ConfigContextFilterSet`?

In general please be sure you're following all recommendations at https://nautobot.readthedocs.io/en/latest/development/extending-models/ for this extension.

Reply:
Also `ConfigContextQuerySet` and `ConfigContextModelQuerySet`